app/controllers/coronavirus_form/additional_product_controller.rb

```diff
@@ -1,36 +1,30 @@
 # frozen_string_literal: true
 
 class CoronavirusForm::AdditionalProductController < ApplicationController
-  before_action :check_first_question_answered, only: :show
-
-  def show
-    render "coronavirus_form/#{PAGE}"
-  end
+  before_action :check_first_question_answered, except: :submit
 
   def submit
     additional_product = strip_tags(params[:additional_product]).presence
 
     invalid_fields = validate_radio_field(
-      PAGE,
+      controller_name,
       radio: additional_product,
     )
 
     if invalid_fields.any?
       flash.now[:validation] = invalid_fields
-      render "coronavirus_form/#{PAGE}"
+      render controller_path
     elsif additional_product == I18n.t("coronavirus_form.questions.additional_product.options.option_yes.label")
-      redirect_to controller: "coronavirus_form/medical_equipment_type", action: "show"
+      redirect_to medical_equipment_type_path
```

Comment:
Nitpick: The vulnerable people form is using the `_url` helper rather than `_path`. There's an explanation for why in the [PR description](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/199)

Reply:
Interesting, thanks 👍